src/templates/multimedia_queue.html

```diff
@@ -12,9 +12,6 @@
         <p class="text-error hqm-error hide">
             <span class="label label-important">There were errors.</span>
         </p>
-        <a href="#" class="btn btn-inverse btn-mini pull-right hqm-remove">
```

Comment:
Seemed clutter-y in a rather small dialog - you can already select a new image to overwrite the first one, or close the dialog altogether.